pbs_light.go

```diff
@@ -193,9 +194,15 @@ type cookieSyncResponse struct {
 	BidderStatus []*pbs.PBSBidder `json:"bidder_status"`
 }
 
-func cookieSync(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
-	mCookieSyncMeter.Mark(1)
-	userSyncCookie := pbs.ParsePBSCookieFromRequest(r, &(hostCookieSettings.OptOutCookie))
+type cookieSyncDeps struct {
+	syncers      map[openrtb_ext.BidderName]usersyncers.Usersyncer
+	optOutCookie *config.Cookie
+	metric       metrics.Meter
```

Comment:
I was going to suggest that metric be a struct so additional metrics could be added in later if needed, but on second thought we can probably change that easily enough if and when more metrics are desired.

Reply:
ok... let me know if you change your mind. I don't have strong feelings either way... but I kept it simple because we don't have any immediate need for more metrics here, and I suspect that #302 will get merged and start to absorb the metrics code before we do.